### ISSUE TITLE
Will the framework continue to be maintained?

### ISSUE THREAD

**Issue by lly0414** [lusaxweb/vuesax-next#80]
Your design is really great, I like it very much, but it seems that the project has not been submitted for two months. Will the framework continue to be maintained? I want to use it all the time in my own projects

**Comment by akirarika** [comment]
Why not?

If you're worried that the project won't continue, maybe you can consider donating or helping author develop

**Comment by lly0414** [comment]
Okay, I was a little worried because I saw that the last update was a long time ago 🙈

**Comment by a0m0rajab** [comment]
@luisDanielRoviraContreras  :) I think this question is for you.

**Comment by luisDanielRoviraContreras** [comment]
Of course we want to continue maintaining and improving the framework

what happens is that we are in commercial projects that take us all the time

when we have time we will give it to vuesax 4

I hope you understand and if you want to help or donate to improve the framework, you are welcome

we hope to be more active when finishing those commercial projects

thank you

**Comment by lly0414** [comment]
I can use it with confidence after getting such a reply. Of course, I hope I have the opportunity to contribute my own code.

**Comment by Heartnett** [comment]
### TLDR;
The authors are too busy with paid work. Contribute or throw money at the project and the authors may or may not continue to work on vuesax.

Conclusion: **Don't use vuesax project for production.**

> Of course we want to continue maintaining and improving the framework
> 
> what happens is that we are in commercial projects that take us all the time
> 
> when we have time we will give it to vuesax 4
> 
> I hope you understand and if you want to help or donate to improve the framework, you are welcome
> 
> we hope to be more active when finishing those commercial projects
> 
> thank you

**Comment by abublihi** [comment]
please make a paid version of the project, I will be the first one to buy it if it supports RTL.
